radix-engine-interface/src/events/mod.rs

```diff
@@ -0,0 +1,13 @@
+use crate::api::types::{Hash, NodeModuleId, RENodeId};
+use crate::ScryptoSbor;
+
+/// Identifies a specific event schema emitter by some emitter RENode.
+///
+/// This type is an identifier uses to identify the schema of events emitted by an RENode of some
+/// [`RENodeId`]. With this identifier, the schema for an event can be queried, obtained, and with
+/// it, the SBOR encoded event data can be decoded and understood.
+///
+/// It is important to note that application events are always emitted by an RENode, meaning that
+/// there is always an emitter of some [`RENodeId`].
```

Comment:
Is this not taking into consideration events emitted by functions yet?

Reply:
Not at the current moment of time. This will change with my next PR where we introduce Schemas into the mix.